Code/Framework/AzCore/AzCore/IO/Streamer/BlockCache.h

```diff
@@ -23,6 +23,11 @@ namespace AZ
 {
     namespace IO
     {
+        class RequestPath;
+        namespace Requests {
+            struct ReadData;
+        }
```

Comment:
coding style: The opening curly brace for the namespace should be on it's own line

Reply:
+1